app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinPresenter.kt

```diff
@@ -41,6 +41,13 @@ class NewCreatePinPresenter(
     private var pinMode = PinMode.CREATE
     private var navigateBackOnBackPressed = false
 
+    override fun attach(view: NewCreatePinContract.View) {
+        super.attach(view)
+        if (onboardingInteractor.currentFlow == OnboardingFlow.CreateWallet) {
+            view.showUiKitSnackBar(messageResId = R.string.auth_create_wallet_introduction)
+        }
```

Comment:
that's not looking as a best solution but it's not critical. 
the interactor should not persist the current state

Reply:
yeah, looks like a repository for me. i'll try and create a ticket as a tech-debt for this